spec/tbd_tests_spec.rb

```diff
@@ -1979,6 +1979,12 @@
       next unless id == "Entryway  Wall 5"
       expect(surface[:heatloss]).to be_within(0.01).of(13.96)
     end
+
+    #out = JSON.pretty_generate(io)
+    #out_path = File.dirname(__FILE__) + "/../json/tbd_minimal_KHI.out.json"
+    #File.open(out_path, "w") do |out_path|
+    #  out_path.puts out
+    #end
```

Comment:
The final version of the _io_ object may be tweaked a few times over the next 48 hours. For now, let's stick its current state.